hangman_lunch.rb

```diff
@@ -0,0 +1,161 @@
+#Lila and Corinne
+
+# works through lunch, first req of dinner (not penalizing repeat guesses on same letter)
+
+require "colorize"
+
+class Hangman
+	def initialize
+
+		# library
+		@words = ["orca", "cheese", "cake", "pumpkin", "sun", "apple", "cloud", "paint", "bird", "travel", "lollipop", "ada"]
```

Comment:
We hadn't learned about constants yet at this point - but if you went back and refactored this would be a good thing to update